Onboarding MSI for daily autogen

serhangursoy-microsoft opened the pull request.

jorgecotillo commented on schemas/2015-08-31-preview/Microsoft.ManagedIdentity.json:
```diff
@@ -7,16 +7,6 @@
     "userAssignedIdentities": {
       "type": "object",
       "properties": {
-        "name": {
```
Are these updates result of running https://github.com/Azure/azure-resource-manager-schemas/actions/workflows/generate-types.yml on your branch?

jorgecotillo replied:
nvm, it is.

jorgecotillo reviewed (changes requested):
Can you please the tests locally,, seems like your update is breaking existing tests: https://github.com/Azure/azure-resource-manager-schemas#runing-unit-tests

serhangursoy-microsoft commented:
Updated the PR. Changed/updated existing test cases. And added two more for newly generated versions. 
Tested locally now - seems to be working as expected
